server/cluster/cluster.go

```diff
@@ -1291,10 +1304,13 @@ func (c *RaftCluster) processRegionHeartbeat(ctx *core.MetaProcessContext, regio
 	}
 
 	if saveKV || needSync {
-		select {
-		case c.changedRegions <- region:
-		default:
-		}
+		ctx.SyncRegionRunner.RunTask(
+			regionID,
+			ratelimit.SyncRegionToFollower,
+			func(context.Context) {
+				c.changedRegions <- region
```

Comment:
maybe we need a metric about `len(c.changedRegions)`

Reply:
Sorry, I don't understand. What is its purpose?